app/src/main/java/de/westnordost/streetcomplete/quests/parking_fee/AddParkingFee.java

```diff
@@ -0,0 +1,37 @@
+/*
+* This quest was generated by the StreetComplete QuestCreator (https://github.com/ENT8R/StreetCompleteQuestCreator)
+*/
+
+package de.westnordost.streetcomplete.quests.parking_fee;
+
+import android.os.Bundle;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddParkingFee extends SimpleOverpassQuestType
+{
+	@Inject public AddParkingFee(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters() { return "nodes, ways with amenity=parking and access!=private and !fee"; }
```

Comment:
Perhaps use access!~private|no in case someone tags parking lot's access with "no"?

Reply:
Or better use ```access ~ yes|customers|public``` like @matkoniecz said in https://github.com/westnordost/StreetComplete/pull/825#issuecomment-362563539 ?

Reply:
Yes, even better.

Reply:
Ah wait, that would mean that parking lots without any access tagged would not be shown. Hmm, yeah, I guess, that is okay, if there is another quest that asks whether it is public or what else that becomes before this one in priority.

Reply:
"if there is another quest" AFAIK we have ticket open for that.

Reply:
Yes, it's #100. Should I also implement this?

Reply:
Given that this issue is open it probably would be welcomed...

Reply:
You can, but you do not have to to get this PR merged.